src/snowflake/snowpark/file_operation.py

```diff
@@ -282,3 +284,66 @@ def put_stream(
         result_meta = cursor.description
         put_result = result_set_to_rows(result_data, result_meta)[0]
         return PutResult(**put_result.asDict())
+
+    def get_stream(
+        self,
+        stage_location,
+        *,
+        parallel: int = 10,
+        statement_params: Optional[Dict[str, str]] = None,
+        decompress: bool = False,
+    ) -> IO[bytes]:
+        """Downloads the specified files from a path in a stage and expose it through a stream.
+
+        Args:
```

Comment:
Add some docstring here? I think it would be good to show proper usage of using `with`.

Reply:
a doctest will be better

Reply:
Oh I meant doctest 🤦‍♂️ Not sure why I typed docstring LMAO